### PR TITLE
v6.1.0

### PR DESCRIPTION
**Pull request by Dougley**
Please refer to the [changelog](https://github.com/TheSharks/WildBeast/blob/experimental/CHANGELOG.md) for a full list of changes

### CODE REVIEW

File: src/commands/meme.js [hunk 1]
```diff
@@ -1,76 +1,47 @@
-const meme = {
```
Comment (linuswillner) [inline src/commands/meme.js]: Need to update the docs to reflect this.

File: src/commands/request.js [hunk 2]
```diff
@@ -36,40 +36,38 @@ module.exports = {
             }
           } else {
             resolveTracks(suffix).then(tracks => {
```
Comment (linuswillner) [inline src/commands/request.js]: My suggestion for `join-voice` applies here as well.

File: src/commands/urbandictionary.js [hunk 3]
```diff
@@ -35,9 +35,10 @@ module.exports = {
                 }
               })
             } else {
-              msg.channel.createMessage(`<@${msg.author.id}>, ${suffix}: This word is so screwed up, even Urban Dictionary doesn't have it in its database`)
+              msg.channel.createMessage(`<@${msg.author.id}>, ${suffix}: This word is so screwed up, even UrbanDictionary doesn't have it in its database`)
```
Comment (linuswillner) [inline src/commands/urbandictionary.js]: Name mangling issue here. According to [Wikipedia](https://en.wikipedia.org/wiki/Urban_Dictionary), the name is in fact **Urban Dictionary**.

File: src/internal/logger.js [hunk 4]
```diff
@@ -64,7 +64,8 @@ module.exports = {
     sendToES({
       type: 'command',
       cmd: opts.cmd,
-      full: opts.cmd + ' ' + opts.opts,
+      // full: opts.cmd + ' ' + opts.opts,
```
Comment (linuswillner) [inline src/internal/logger.js]: Commented out code should not be left in the production build.

File: src/internal/version-check.js [hunk 5]
```diff
@@ -7,4 +11,13 @@
...
-})()
+})
+
+async function gitVerify () {
+  const currentHead = await exec('git rev-parse --abbrev-ref HEAD')
+  const remoteRef = await exec(`git rev-parse origin/${currentHead.stdout.trim()}`)
+  const currentRef = await exec('git rev-parse HEAD')
+  const count = await exec(`git rev-list ${currentRef.stdout.trim()}..${remoteRef.stdout.trim()} --count`)
+  if (count.stdout.trim() > 0) global.logger.warn(`You're behind ${count.stdout.trim()} commit(s) compared to remote branch ${currentHead.stdout.trim()}, update recommended`)
+  else global.logger.log('Fully up-to-date, nice!')
```
Comment (linuswillner) [inline src/internal/version-check.js]: Language revision: I suggest using **You're fully up to date!** instead.

Comment (Dougley) [reply src/internal/version-check.js]: Dunno, I like the less serious tone of the current message more.

File: src/commands/urbandictionary.js (hunk 3 above)
Comment (Dougley) [reply src/commands/urbandictionary.js]: Hah, accidentally changed this with my last commit, will fix.

**Pull request merged**